src/getBucketsForNode.ts

```diff
@@ -7,12 +9,18 @@
  *
  * @param node - The node which buckets should be retrieved
  */
-export default function getBucketsForNode(node: Node): string[] {
```

Comment:
I refactored the type parameter in order to remove the externally facing 'Node' type. We should not enforce what kind of DOM implementation a user of these APIs should use, it will give problems with using something like slimdom in a typescript environment (such as the tests)